numpy/ma/tests/test_core.py

```diff
@@ -297,17 +297,28 @@ def test_copy(self):
                         y1._data.__array_interface__)
         self.assertTrue(y1a.mask is y1.mask)
 
-        y2 = array(x1, mask=m)
+        y2 = array(x1, mask=m3)
         self.assertTrue(y2._data.__array_interface__ == x1.__array_interface__)
-        #self.assertTrue( y2.mask is m)
-        self.assertTrue(y2._mask.__array_interface__ == m.__array_interface__)
+        #self.assertTrue( y2.mask is m3)
```

Comment:
And if it is a commented out statement, it should be removed. Frankly, I don't understand what is going on here.